lib/containers/utils.pm

```diff
@@ -86,7 +91,7 @@ sub test_update_cmd {
 
     # podman update was added to v4.3.0
     if ($runtime eq 'podman') {
-        my $version = script_output "podman version | awk '/^Version:/ { print \$2 }'";
+        my $version = get_podman_version()
```

Comment:
```suggestion
        my $version = get_podman_version();
```

Reply:
Ok, here there is something you need to change due to Perl syntax .. you can't declare a variable inside an `if`.
Please do this:
```
my $version;
if ($runtime eq 'podman') {
    $version = get_podman_version();
    if (package_version_cmp($version, '4.3.0') <= 0) {
        record_info("SKIP", "The update command is not supported on podman $version");
        return;
    }
} ....
```

or if you wanna simplify it:
```
if ($runtime eq 'podman' && package_version_cmp(get_podman_version, '4.3.0') <= 0) {
    record_info("SKIP", "The update command is not supported on podman version");
    return;
}
```